src/App.js

```diff
@@ -1,10 +1,9 @@
-import logo from './logo.svg';
 import './App.css';
-
+import Footer from './components/footer';
 function App() {
   return (
     <div className="App">
-      
+      <Footer></Footer>
```

Comment:
```suggestion
      <Footer />
```
Componentes sin children no tienen que tener open y close tags.